From 623258a6517c8ec1d9e5f41cbdc05205a0e6ee1d Mon Sep 17 00:00:00 2001
From: David Allsopp <david.allsopp@metastack.com>
Date: Thu, 5 May 2022 20:01:44 +0100
Subject: [PATCH 02/24] Merge pull request #11236 from Nymphium/missing-since2

Add missing @since annotation to Gc.eventlog_pause

(cherry picked from commit 77fee6035c25d8a31084dc556ee634e46bb39164)
---
 stdlib/gc.mli | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/stdlib/gc.mli b/stdlib/gc.mli
index b211197fd4..8031eeb8df 100644
--- a/stdlib/gc.mli
+++ b/stdlib/gc.mli
@@ -442,7 +442,10 @@ external eventlog_pause : unit -> unit = "caml_eventlog_pause"
    Traces are collected if the program is linked to the instrumented runtime
    and started with the environment variable OCAML_EVENTLOG_ENABLED.
    Events are flushed to disk after pausing, and no new events will be
-   recorded until [eventlog_resume] is called. *)
+   recorded until [eventlog_resume] is called.
+
+   @since 4.11
+  *)
 
 external eventlog_resume : unit -> unit = "caml_eventlog_resume"
 (** [eventlog_resume ()] will resume the collection of traces in the
@@ -451,7 +454,10 @@ external eventlog_resume : unit -> unit = "caml_eventlog_resume"
    and started with the environment variable OCAML_EVENTLOG_ENABLED.
    This call can be used after calling [eventlog_pause], or if the program
    was started with OCAML_EVENTLOG_ENABLED=p. (which pauses the collection of
-   traces before the first event.) *)
+   traces before the first event.)
+
+   @since 4.11
+  *)
 
 
 (** [Memprof] is a sampling engine for allocated memory words. Every
-- 
2.37.0.rc2

